hermes-console-vue/src/composables/search/useSearch.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { afterEach, describe } from 'vitest';
+import {
+  dummySubscription,
+  secondDummySubscription,
+} from '@/dummy/subscription';
+import { dummyTopic } from '@/dummy/topic';
+import {
+  querySubscriptionsErrorHandler,
+  querySubscriptionsHandler,
+  queryTopicsErrorHandler,
+  queryTopicsHandler,
+} from '@/mocks/handlers';
+import { SearchFilter, useSearch } from '@/composables/search/useSearch';
+import { setupServer } from 'msw/node';
+import { waitFor } from '@testing-library/vue';
+
+describe('useSearch', () => {
+  const server = setupServer(queryTopicsHandler({ topics: [dummyTopic] }));
+
+  afterEach(() => {
+    server.resetHandlers();
+  });
+
+  it('should retrieve topic query results from Hermes API', async () => {
+    // given
+    server.listen();
+
+    // when
+    const { topics, queryTopicsFn, loading, error } = useSearch();
+
+    queryTopicsFn(SearchFilter.NAME, 'query');
```

Comment:
You don't have to name function with 'Fn' suffix. queryTopics/searchTopics/findTopics etc. would be just fine

Reply:
fixed